packages/sdk-js/src/models/step.ts

```diff
@@ -52,6 +52,8 @@ class Step implements StepProps {
     const outputDataType = step.getOutputDataCase();
     console.log("step.getOutput.outputDataType", outputDataType);
     switch (outputDataType) {
+      case avs_pb.Execution.Step.OutputDataCase.OUTPUT_DATA_NOT_SET:
+        return {} as OutputDataProps;
```

Comment:
since the return of this function is `any`, if the outputData is NOT_SET, we can just return `undefined` here, right? Or `null` is better?